analyzer/queries/queries.go

```diff
@@ -337,6 +337,12 @@ var (
     INSERT INTO chain.runtime_related_transactions (runtime, account_address, tx_round, tx_index)
       VALUES ($1, $2, $3, $4)`
 
+	RuntimeAccountUpsert = `
```

Comment:
Nit/optional: RuntimeAccountNumTxsUpsert, so it's clearer what we're doing from the callsite. Also, if this table ever acquires more columns, it's unlikely we'll be updating all of them in a single upsert, so this one will likely stay dedicated to num_txs.